plyer/facades/screenshot.py

```diff
@@ -0,0 +1,62 @@
+'''
+ScreenShot
```

Comment:
According to [Oxford dictionary](https://en.oxforddictionaries.com/definition/screenshot) a "screenshot" is a real word, unlike "FileChooser", therefore it'd be better to go for that. Also, I'd say it's more expected to do:

    from something import Screenshot

instead of:

    from something import ScreenShot

as it will be more confusing in the future. Also, it's inconsistent with the rest of your code, where you use "screenshot" as a single word.